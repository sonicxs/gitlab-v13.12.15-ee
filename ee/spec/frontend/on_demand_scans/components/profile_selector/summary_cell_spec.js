import { mount } from '@vue/test-utils';
import OnDemandScansProfileSummaryCell from 'ee/on_demand_scans/components/profile_selector/summary_cell.vue';

describe('OnDemandScansProfileSummaryCell', () => {
  let wrapper;

  const createFullComponent = () => {
    wrapper = mount(OnDemandScansProfileSummaryCell, {
      propsData: {
        label: 'Row Label',
        value: 'Row Value',
      },
    });
  };

  afterEach(() => {
    wrapper.destroy();
  });

  it('renders properly', () => {
    createFullComponent();

    expect(wrapper.element).toMatchSnapshot();
  });
});
