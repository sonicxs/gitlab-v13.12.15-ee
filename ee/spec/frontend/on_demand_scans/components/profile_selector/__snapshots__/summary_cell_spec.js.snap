// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OnDemandScansProfileSummaryCell renders properly 1`] = `
<div
  class="col-md-6"
>
  <div
    class="row gl-my-2"
  >
    <div
      class="col-md-4"
    >
      Row Label:
    </div>
     
    <div
      class="col-md-8"
    >
      <strong>
        Row Value
      </strong>
    </div>
  </div>
</div>
`;
